src/screens/CourseScreen/index.tsx

```diff
@@ -110,6 +126,13 @@ export const CourseScreen = () => {
 			>
 				<ChatBubbleIcon sx={{ color: '#e9ecef' }} />
 			</Button>
+			<FeedbackModal
+				opneModal={openModal}
+				handleOpenModal={handleOpenModal}
+				feedback={feedback}
+				setFeedBack={setFeedBack}
+				feedbackSubmitHandler={feedbackSubmitHandler}

```

Comment:
delete this prop since we agreed that the state will live in the modal itself and the submit function too since the parent component is not interseted in it